assets/js/atomic/blocks/product-elements/add-to-cart-form/style.scss

```diff
@@ -0,0 +1,7 @@
+.woocommerce.wp-block-add-to-cart-form {
```

Comment:
I think that we should avoid custom CSS rules, especially if we will implement Global Styles.

Reply:
See: https://github.com/woocommerce/woocommerce-blocks/pull/8482#discussion_r1114513682